### ISSUE TITLE
v 0.67 - Markdown Enabled - Issues:

### ISSUE THREAD

**Issue by ahartlaub** [RocketChat/Rocket.Chat#11310]
### Description:
When Markdown is enabled (under General->Messages) - the following do not work:
@mentions
Announcement text

### Steps to reproduce:
1. Change markdown from Original to Markdown (in the markdown menu).  Attempt to do a @mention or set a room announcement.

### Expected behavior:

Mentions properly alert the users.  Announcement text is visible.

### Actual behavior:

Mentions do not work.  Announcement bar is blue with no text.

![image](https://user-images.githubusercontent.com/11774580/42173258-17084efa-7de4-11e8-88cf-efd67c4ed3a9.png)

### Server Setup Information:

- Version of Rocket.Chat Server: 0.67
- Operating System:  Ubuntu 16.04
- Deployment Method: Docker
- Number of Running Instances: 1
- DB Replicaset Oplog: 
- NodeJS Version: 
- MongoDB Version:

**Comment by kajisaap** [comment]
Yes, same observation but our version is 0.66.0 and we are using multiple RC instances.

For the announcement banner, we tried to fix via custom CSS but in vain. It used to work in previous versions.

```css
.rc-old .announcement p {
   width: auto !important;
}
```

`@` mentions are not stylized and does not notify when it is leading the conversation - if you pad it with anything even one space, it works.

**Comment by vynmera** [comment]
This should fix it: #11357
